src/main/scala/is/hail/utils/richUtils/RichPairRDD.scala

```diff
@@ -57,4 +57,8 @@ class RichPairRDD[K, V](val rdd: RDD[(K, V)]) extends AnyVal {
 
   def toOrderedRDD[PK](implicit kOk: OrderedKey[PK, K], vct: ClassTag[V]): OrderedRDD[PK, K, V] =
     OrderedRDD[PK, K, V](rdd, None, None)
+
+  def shuffleTo(p: Partitioner)(implicit kct: ClassTag[K], vct: ClassTag[V]): ShuffledRDD[K, V, V] = {
```

Comment:
That's the one! I tried to find that functionality but didn't guess the right names.